### PR TITLE
step1 fork and python environment

### PR DESCRIPTION
**Pull request by 4383**
explain how to setup fork and clone and how to setup python environment

### CODE REVIEW

File: _drafts/contribute-to-ansible.markdown [hunk 1]
```diff
@@ -111,13 +111,93 @@ Hovewer you can [follow topics projects](https://github.com/ansible/ansible/proj
...
-    a. Récupérer les sources
-        . Forker le projet
-        . checkouter en local
-        . configurer l'upstream (avec comment update le fork)
-        . se placer sur la bonne branche
 
-    b. Créer un environnement python (pyvenv)
+#### a. Setup your fork
+
+First, go to the [ansible github official page](https://github.com/ansible/ansible) and click on the fork button.
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown:117]: Take some snapshots for final article it you can

File: _drafts/contribute-to-ansible.markdown [hunk 2]
```diff
@@ -111,13 +111,93 @@ Hovewer you can [follow topics projects](https://github.com/ansible/ansible/proj
...
+Wait a few moments and now your fork appear in your github account.
+
+Now prepare your local environment.
+
+```sh
+$ git config --global user.name "<your name>"
+$ git config --global user.email "<your-email@example.com>"
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: You should need to add the config push.default method (simple or matching)

File: _drafts/contribute-to-ansible.markdown [hunk 3]
```diff
@@ -111,13 +111,93 @@ Hovewer you can [follow topics projects](https://github.com/ansible/ansible/proj
...
+
+Ansible come with a lots of development dependencies, you can install all or just a subset
+in concordance with your development purpose.
+
+If you work on documentation all development dependencies are presents in `./docsite_requirements.txt`.
+```sh
+$ pip install -r ./docsite_requirements.txt
+```
+
+For working on bugfix, tests, or new features you need to install specific requirements:
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: To work on bugfix.....

**Review by dj4ngo — CHANGES_REQUESTED** [review]

**Comment by 4383** [comment]
@dj4ngo fix review done!

**Review by dj4ngo — APPROVED** [review]

**Pull request merged**